app/api/views/sudo.py

```diff
@@ -0,0 +1,27 @@
+from flask import jsonify, g, request, make_response
+from sqlalchemy_utils.types.arrow import arrow
+
+from app.api.base import api_bp, require_api_auth
+from app.db import Session
+
+
+@api_bp.route("/sudo", methods=["PATCH"])
```

Comment:
should we add this endpoint to api.md ?

Reply:
Added. We really need at some point to generate the docs automatically from the methods.